app/views/providers/about_the_financial_assessments/show.html.erb

```diff
@@ -1,22 +1,30 @@
 <%= page_template(page_title: t('.title')) do %></h1>
   <p class="govuk-body"><%= t '.sign_in_para' %></p>
 
-  <p class="govuk-body"><%= t '.share_bank_accounts_para' %></p>
-
-  <h2 class="govuk-heading-m"><%= t '.list.title' %></h2>
-
   <p class="govuk-body">
     <%= list_from_translation_path 'providers.about_the_financial_assessments.show.list' %>
   </p>
 
+  <h2 class="govuk-heading-m"><%= t '.section_1.title' %></h2>
+
+  <p class="govuk-body"><%= t '.section_1.detail_use' %></p>
+
+  <p class="govuk-body"><%= t '.section_1.sign_in_save' %></p>
+
   <h2 class="govuk-heading-m">
-    <%= t '.section_1.heading' %>
+    <%= t '.section_2.heading' %>
   </h2>
 
   <p>
-    <%= t '.section_1.text' %>
+    <%= t '.section_2.text' %>
   </p>
 
+  <%= check_answer_email_link(
+        name: :email,
+        url: providers_legal_aid_application_update_email_address_path(anchor: :email), # currently not using anchor may need fo rnew page
```

Comment:
I don't understand this comment `currently not using anchor may need fo rnew page`
This link IS using an anchor.
Is a full stop missing?

Reply:
I meant to delete this